keycloak/keycloak-admin-ui
# User Federation: populate Kerberos required and cache settings data

## Motivation
Related JIRA issues:
https://issues.redhat.com/browse/APPDUX-655

Design spec:
https://marvelapp.com/prototype/ecjc94b/screen/72098368

## Brief Description
Fetches and displays settings data for a specific individual provider card that is clicked in the user federation card view. This PR does **not** cover save nor the showing/hiding of UI components based on selections. These will be handled in future PRs. This PR only covers pulling in the data and getting to the settings page from the cards.

## Verification Steps
1. Click a card in the user fed card view.
2. Verify that all settings match those that have been configured for that provider (presumably, the ones configured and saved in the old admin UI).
3. Change the values and verify that the changes are reflected in the new UI.

## Checklist:

## Additional Notes
Required settings:
![kerberos-req-settings](https://user-images.githubusercontent.com/39063664/100771949-016a7b80-33cd-11eb-941c-fcb73dd76ac9.png)

Cache settings:
![kerberos-cache-settings](https://user-images.githubusercontent.com/39063664/100771929-fc0d3100-33cc-11eb-965c-45e4a846761a.png)
